<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openCSV</class>
 <widget class="QDialog" name="openCSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open CSV File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourcelineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>&amp;Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="filegroupBox">
       <property name="title">
        <string>File Format</string>
       </property>
       <widget class="QCheckBox" name="rownamecheckBox">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Row Names</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="headercheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>58</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>header</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="standardgroupBox">
       <property name="title">
        <string>Standard</string>
       </property>
       <widget class="QRadioButton" name="EUradioButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>57</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Europe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="USradioButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>43</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>USA</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>&amp;OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>openCSV</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>openCSV</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
